Gemfile

```diff
@@ -87,3 +87,4 @@ group :test do
   # get test coverage info on codeclimate
   gem "codeclimate-test-reporter", group: :test, require: nil
 end
+gem "nokogiri", ">= 1.6.7.2"
```

Comment:
This is disappointing. The docs say it just does `bundle update`. I didn't want to see updates to the Gemfile.